### ISSUE TITLE
Loss used for pretrained perceptual models? 

### ISSUE THREAD

**Issue by samaonline** [EPFL-VILAB/XTConsistency#3]
It's great that network architectures of pretrained perceptual models are in transfers.py. You mentioned "some of the pretrained models were trained using L2 loss". Do you have a list of loss used for different tasks? Or are they the same as those defined in [taskonomy](https://github.com/StanfordVL/taskonomy/tree/master/taskbank)?

**Comment by aserety** [comment]
All the pretained perceptual models used a L2 loss. We trained the rgb->{depth, normals, reshading} models with L1. The loss used depends on whether it is a perceptual network or not, rather than on the task. The wording in the readme may have been a bit confusing, sorry about that, we'll update it.

**Comment by samaonline** [comment]
Thanks for the clarification!
